### PR TITLE
use @celo/celo-devchain for sdk tests needing a devchain

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Previously to run tests in sdk such as cli and contractkit that needed a mock chain to test against it was neccessary to build everything by hand each time. and required running ascript in protocol. this will not be possible once ck is moved out. and for now just speeds up everything. 

### Other improvements

Note before this i have not been able to run ck locally for a little while and now i can so major win.

### Related issues

This is part of the move to untangle contractkit from protocol so that sdks can be move to a new repo. the other remaining are 

* replacing script that builds contractkits generated folder which contains Typescript Contracts with importing those same files from @celo/abis 
* remove function used in contractkit tests that is imported from protocol but also duplicated in contractkit itself 
* remove MetaTransactionWallet test which rely on files from protocol
* modify SortedOracles test which have same issue

### Backwards compatibility

n/a

### Documentation
n/a

### CODE REVIEW

File: packages/sdk/contractkit/src/test-utils/setup.global.ts [hunk 1]
```diff
@@ -8,8 +8,9 @@ import * as path from 'path'
 // If there is not, then your editor probably deleted it automatically.
 
 export default async function globalSetup() {
-  console.log('\nstarting ganache...')
```
Comment (shazarre) [inline packages/sdk/contractkit/src/test-utils/setup.global.ts:11]: you don't want to keep the log here?

**Review by shazarre — APPROVED** [review]
left one minor comment, otherwise looks good to me

**Comment by soloseng** [comment]
Not sure if intended, but `transaction-uri` is not being tested in CI

**Comment by aaronmgdr** [comment]
> Not sure if intended, but `transaction-uri` is not being tested in CI

ACK!

**Pull request merged**